bom/pom.xml

```diff
@@ -266,7 +266,7 @@ THE SOFTWARE.
       <dependency><!-- StAX implementation. See JENKINS-2547. -->
         <groupId>org.codehaus.woodstox</groupId>
         <artifactId>wstx-asl</artifactId>
-        <version>3.2.9</version>
+        <version>4.0.0</version>
       </dependency>
```

Comment:
This dependency is slated for removal pending the tasks described in [#5604 (comment)](https://github.com/jenkinsci/jenkins/pull/5604#issuecomment-873539241). Most of those tasks are already done, so I'd rather see us take that across the finish line rather than risk breaking something again (the last time I changed this dependency, I caused a regression). Let me know if you'd like to help with these tasks.